src/analysis/locate.ml

```diff
@@ -310,6 +310,16 @@ let load_cmt ~config comp_unit ml_or_mli =
       Ok (source_file, cmt_infos)
   | None -> Error ()
 
+let find_declaration_uid ~env ~decl_uid path =
```

Comment:
I find that name to be quite vague, especially since we give `decl_uid` as an input.
How about renaming it to `scrape_alias` or something of the sort?

Reply:
Done in 31bbe7408e0767d4334955198fdb9ec377af5fe8